azure/services/vnetpeerings/vnetpeerings_test.go

```diff
@@ -65,6 +66,30 @@ var (
 		RemoteResourceGroup: "group1",
 		SubscriptionID:      "sub1",
 	}
+	fakePeeringHubToSpoke = VnetPeeringSpec{
+		PeeringName:               "hub-to-spoke",
+		SourceVnetName:            "hub-vnet",
+		SourceResourceGroup:       "hub-group",
+		RemoteVnetName:            "spoke-vnet",
+		RemoteResourceGroup:       "spoke-group",
+		SubscriptionID:            "sub1",
+		AllowForwardedTraffic:     to.BoolPtr(true),
```

Comment:
```suggestion
		AllowForwardedTraffic:    pointer.Bool(true),
```
I think we want to switch to the `"k8s.io/utils/pointer"` library